index.js

```diff
@@ -1,4 +1,13 @@
 'use strict'
 
-const { sagaTestEngine } = require('./src')
-module.exports = sagaTestEngine
+const {
+    collectPuts,
+    collectCalls,
+    collectCallsAndPuts,
+    createSagaTestEngine
+} = require('./src')
+
+exports.default = collectPuts
+exports.collectCalls = collectCalls
+exports.collectCallsAndPuts = collectCallsAndPuts
+exports.createSagaTestEngine = createSagaTestEngine
```

Comment:
I thought some more about this and I think backwards compatibility maybe is something we should sacrifice here for a better API. Especially since this will be a major semver bump anyway.

Proposed:
```js
module.exports = {
  default: createSagaTestEngine,
  createSagaTestEngine,
  collectCalls,
  collectCallsAndPuts,
}
```

Any thoughts on how you would prefer this? We can't really perfectly preserve the existing API anyway.

Also, I wonder if we should expose `createSagaTestEngine` as something like `makeEffectCollector`, since it creates functions like `collectPuts` et al.